content/en/tracing/guide/setting_up_apm_with_kubernetes_service.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Setting up APM with Kubernetes Service
+kind: guide
+further_reading:
+- link: "/containers/kubernetes/apm/"
+  tag: "Documentation"
+  text: "Set up trace collection"
+- link: "/containers/cluster_agent/admission_controller"
+  tag: "Documentation"
+  text: "Admission Controller"
+---
+
+## Overview
+
+There are 3 main ways the Datadog Tracers send their data to the Datadog Agent for the proper handling. Those are: Host IP, Unix Domain Socket (UDS), and the Kubernetes service. These options are all setup in a way to ensure the APM data the application pod sends arrive to the Datadog Agent pod on the same node. This strategy is meant to properly balance the traffic and ensure the correct tagging of your data.
+
+Datadog recommends to utilize the UDS strategy to send data. However, the Kubernetes service can be used as an alternative option when the `hostPath` volumes required for UDS and the `hostPort` ports required for the Host IP method are not available.
+
+## Service setup
+
+In Kubernetes you can take advantage of [services][1] to help route traffic to downstream pods. One limitation of the default service configuration is that this routes the traffic to the downstream pods at random. In Kubernetes 1.22 the [Internal Traffic Policy feature][2] was added, giving the option to set the configuration `internalTrafficPolicy: Local` on the service. When set this directs the traffic from an application pod to the service's downstream pod *on the same node*. Maintaining that strategy of routing the APM data to the Datadog Agent on the same node.
+
+This service is created for you automatically in our Datadog Helm Chart and Datadog Operator on Kubernetes clusters with version 1.22.0 or above. You additionally need to enable the APM port option for your Agent with the below configuration.
+
+### Agent Configuration
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+```yaml
+datadog:
+  apm:
+    portEnabled: true
+```    
+
+{{% /tab %}}
+{{% tab "Operator" %}}
+
+```yaml
+apiVersion: datadoghq.com/v2alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  #(...)
+  features:
+    apm:
+      enabled: true
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+## Application configuration
+To take advantage of the service the Datadog Tracer needs the `DD_AGENT_HOST` environment variable set relative to the created service. 
```

Comment:
It looks like if you're using Admission Controller, you may not really interact with any kind of `DD_AGENT_HOST` env variable (seems like it's set for you) so let's not mention it here.

```suggestion
You can configure your application to use the Kubernetes service manually, or by using the [Cluster Agent Admission Controller][3].
```